src/components/the-button.ts

```diff
@@ -226,8 +226,12 @@ export class TheButton extends BotComponent {
             }
             // check to see if the user has pressed it within the last 24 hours
             if(Date.now() - scoreboard[interaction.user.id].last_press <= PRESS_TIMEOUT) {
+                // ~~x converts the float x to an integer
+                // next_possible is the unix-time for the next possible button press
+                const next_possible = ~~((scoreboard[interaction.user.id].last_press + DAY) / 1000)
```

Comment:
Can we reuse `PRESS_TIMEOUT` here (just in case it's ever changed to be something other than a `DAY`)

Reply:
Yup, good idea. 
Done.

Reply:
Thanks!

Reply:
Ah whoopsie, also forgot a semicolon